teammapper-backend/src/map/utils/clientServerMapping.ts

```diff
@@ -55,6 +55,31 @@ const mapMmpMapToClient = (
   }
 }
 
+const mergeClientNodeIntoMmpNode = (
```

Comment:
I'm not sure the spread operator works here considering that the client node has different keys to the server node (i.e. `colors.name` instead of `colorsName`)? I also wanted to give explicit priority to the client node as that's where the diffs are coming from.

Reply:
ah ok sorry nvmd